javaslang/src/main/java/javaslang/collection/CharSeq.java

```diff
@@ -2792,6 +2792,7 @@ public int parseInt(int radix) {
      * @throws NumberFormatException If this {@code CharSeq} does not contain a parsable unsigned int.
      * @since 2.1.0
      */
+    @GwtIncompatible
     public int parseUnsignedInt() {
         return Integer.parseUnsignedInt(back);
```

Comment:
we could make it GWT compatible, but I doubt it would be very useful from javascript:
```java
public int parseUnsignedInt() {
    int result = Integer.parseInt(back);
    if (result < 0) { throw new NumberFormatException(back + " is not unsigned!"); }
    return result;
}
```